hack/chains/README.md

```diff
@@ -19,8 +19,9 @@ For CRC:
 
 For cluster-bot (in slack):
 
-    launch 4.9.15 # or launch latest ?
+    launch 4.9.15 $PLATFORM # or launch latest ?
     # ...and follow the instructions to authenticate once it's ready
+    # Use one of the following tested platforms: ovirt, openstack, azure, aws
```

Comment:
1. Did you want to mention which one is the most likely to work with the chains demos?
2. Every time I tried launch latest it would fail, so I'm not sure if it's useful. Did you get that to work ever? If not, then maybe we should remove it

Reply:
Yes and removed the launch command.